t.css

```diff
@@ -0,0 +1,5 @@
+a {}
```

Comment:
Expected empty line before at-rule.

```suggestion

@media {}```